tests/integration/config/plugins/integration-tests/data/tests/tests_spawn_limits.txt

```diff
@@ -0,0 +1,254 @@
+	"player reputation" 1e9
+	"fine" 0
+	"attitude toward"
+		"Test Government 2" -1
+
+government "Test Government 1 C"
+	swizzle 3
+	color 1 1 1
+	"player reputation" 1e9
+	"fine" 0
+	"attitude toward"
+		"Test Government 2" -1
+
+government "Test Government 2"
+	swizzle 4
+	color 1 1 1
+	"player reputation" 1e9
+	"fine" 0
+	"attitude toward"
+		"Test Government 1 A" -1
+		"Test Government 1 B" -1
+		"Test Government 1 C" -1
+
+mission "Board 1 A"
+	assisting
+	repeat
+	source
+		government "Test Government 1 A"
+	on offer
+		"test: Test Government 1 A boarded" ++
+	destination "Earth"
+
+mission "Board 1 B"
+	assisting
+	repeat
+	source
+		government "Test Government 1 B"
+	on offer
+		"test: Test Government 1 B boarded" ++
+	destination "Earth"
+
+mission "Board 1 C"
+	assisting
+	repeat
+	source
+		government "Test Government 1 C"
+	on offer
+		"test: Test Government 1 C boarded" ++
+	destination "Earth"
+
+mission "Board 2"
+	assisting
+	repeat
+	source
+		government "Test Government 2"
+	on offer
+		"test: Test Government 2 boarded" ++
+	destination "Earth"
+
+ship "Kasichara A'awoj" "Big Ship"
+	outfits
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Systems Core (Small)"
+		"Large Heat Shunt" 2
+		"Small Heat Shunt" 3
+		"Fuel Processor" 2
+		"Luxury Accommodations"
+		"Microbot Defense Station" 23
+		"Thermal Repeater Rifle" 548
+
+fleet "Disabled 2 Test Fleet"
+	government "Test Government 2"
+	cargo 0
+	personality staying derelict
+	variant 60
+		"Big Ship"
+
+
+
+test "Spawn Limit Tests"
+	status active
+	description "Tests if random interval and system entry fleet spawn limits work."
+	sequence
+		inject "Spawn Limits Test"
+		call "Load First Savegame"
+		watchdog 12000
+		call "Depart"
+		# Wait for 10 seconds to drift away from the planet.
+		apply
+			"test: steps to wait" = 10 * 60
+		# empty input to make time pass
+		input
+
+		# Board first ship.
+		input
+			command board
+		apply
+			"test: steps to wait" = 3 * 60
+		label wait1
+		apply
+			"test: steps to wait" = "test: steps to wait" - 1
+		branch wait1
+			"test: steps to wait" > 0
+
+		# Board second ship.
+		input
+			command board
+		apply
+			"test: steps to wait" = 3 * 60
+		label wait2
+		apply
+			"test: steps to wait" = "test: steps to wait" - 1
+		branch wait2
+			"test: steps to wait" > 0
+
+		# Board third ship.
+		input
+			command board
+		apply
+			"test: steps to wait" = 3 * 60
+		label wait3
+		apply
+			"test: steps to wait" = "test: steps to wait" - 1
+		branch wait3
+			"test: steps to wait" > 0
+
+		# Hopefully board no other ships.
+		input
+			command board
+		apply
+			"test: steps to wait" = 3 * 60
+		label wait4
+		apply
+			"test: steps to wait" = "test: steps to wait" - 1
+		branch wait4
+			"test: steps to wait" > 0
+
+		assert
+			not "test: Test Government 1 A boarded"
+		assert
+			"test: Test Government 1 B boarded" == 1
+		assert
+			"test: Test Government 1 C boarded" == 1
+		assert
+			"test: Test Government 2 boarded" == 1
```

Comment:
You can do all of these in the same assert
If you want to know which failed just apply them into "test: <name>"

Reply:
I don't understand. What does this mean?

>  If you want to know which failed just apply them into "test: "

I want to know which one failed, and I am unaware of any other way.

Reply:
Alr lemme suggest a change that'll do it